### PR TITLE
Add centos 7.2 templates

### PR DESCRIPTION
**Pull request by armiller**
These should be correct but I was not able to get them to build on the workstations. It was hanging on SSH.

\cc @ramereth

### CODE REVIEW

**Comment by ramereth** [comment]
@armiller mind removing the non-openstack/ganeti template? I'd rather we stick to ones we actually manage ourselves here.

**Comment by ramereth** [comment]
@armiller you're also missing the kickstart files for the new version.

**Comment by armiller** [comment]
Can i copy those from 7.1?

**Comment by ramereth** [comment]
@armiller yes, but check paths as a few have version numbers in them.

**Comment by armiller** [comment]
@ramereth I still wasn't able to boot due to qemu errors.

**Comment by jordane** [comment]
@armiller you can't have VirtualBox and qemu running at the same time. That's a common problem with the workstations.

File: packer/http/centos-7.2/ks-ganeti.cfg [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+part /boot --fstype="ext4" --size=256
+part swap --fstype="swap" --size=512
+part / --fstype="ext4" --grow --size=100
+auth --enableshadow --passalgo=sha512 --kickstart
+firstboot --disabled
+reboot
+services --disabled=NetworkManager
+user --name=osuadmin --plaintext --password osuadmin
+url --url=http://centos.osuosl.org/7/os/x86_64/
+repo --name=updates --baseurl=http://centos.osuosl.org/7/updates/x86_64/
```
Comment (ramereth) [inline packer/http/centos-7.2/ks-ganeti.cfg]: Let's go ahead and statically set this to `7.2` so it always builds that version.

File: packer/http/centos-7.2/ks-openstack.cfg [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+skipx
+zerombr
+clearpart --all --initlabel
+part / --fstype="ext4" --grow --size=100
+auth --enableshadow --passalgo=sha512 --kickstart
+firstboot --disabled
+reboot
+user --name=centos --plaintext --password centos
+url --url=http://centos.osuosl.org/7/os/x86_64/
+repo --name=updates --baseurl=http://centos.osuosl.org/7/updates/x86_64/
```
Comment (ramereth) [inline packer/http/centos-7.2/ks-openstack.cfg]: Let's go ahead and statically set this to `7.2` so it always builds that version.

File: packer/http/centos-7.2/ks.cfg [hunk 3]
```diff
@@ -0,0 +1,80 @@
+install
```
Comment (armiller) [inline packer/http/centos-7.2/ks.cfg]: Since we are not adding non openstack or ganeti images I guess this file can be removed?

Comment (ramereth) [inline packer/http/centos-7.2/ks.cfg]: Yes please

**Comment by armiller** [comment]
I built the openstack 7.2 image successfully on util. Is this ready to merge?

**Comment by ramereth** [comment]
:+1:

**Pull request merged**